SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -480,28 +533,43 @@ void InterestFactory::GetActorUserDefinedQueries(const AActor* InActor, const Qu
 	{
 		for (const auto& Child : InActor->Children)
 		{
-			GetActorUserDefinedQueries(Child, LevelConstraints, OutQueries, true);
+			GetActorUserDefinedQueryConstraints(Child, OutFrequencyToConstraints, true);
 		}
 	}
 }
 
-TArray<Query> InterestFactory::GetUserDefinedQueries(const QueryConstraint& LevelConstraints) const
+TArray<Query> InterestFactory::GetNetCullDistanceFrequencyQueries(const QueryConstraint LevelConstraint) const
```

Comment:
LevelConstraint copy argument. Do we want that?

Reply:
made this a reference